grammar/rainerscript.h

```diff
@@ -26,6 +26,9 @@
 #include <regex.h>
 #include "typedefs.h"
 
+#ifdef _AIX
+#define var var_tt
```

Comment:
Yes I understand. So since renaming variables across rsyslog will be irrespective of OS check, I think it is better to open a separate issue/bug to drop these changes.